resources/zib/zib-TobaccoUse.xml

```diff
@@ -0,0 +1,230 @@
+        <comment value="TobaccoUseStatus" />
+      </mapping>
+    </element>
+    <element id="Observation.note">
+      <path value="Observation.note" />
+      <max value="1" />
+    </element>
+    <element id="Observation.note.text">
+      <path value="Observation.note.text" />
+      <short value="Comment" />
+      <definition value="Relevant comments on the patient’s use of tobacco." />
+      <alias value="Toelichting" />
+      <mapping>
+        <identity value="zib-tobaccouse-v3.2-2020EN" />
+        <map value="NL-CM:7.2.3" />
+        <comment value="Comment" />
+      </mapping>
+    </element>
+    <element id="Observation.component">
+      <path value="Observation.component" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="code" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.component:typeOfTobaccoUsed">
+      <path value="Observation.component" />
+      <sliceName value="typeOfTobaccoUsed" />
+    </element>
+    <element id="Observation.component:typeOfTobaccoUsed.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="53661000146106" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:typeOfTobaccoUsed.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="TypeOfTobaccoUsed" />
+      <definition value="Type of tobacco the patient uses." />
+      <alias value="SoortTabakGebruik" />
+      <type>
+        <code value="CodeableConcept" />
+      </type>
+      <binding>
+        <strength value="extensible" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.7.2.3--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-tobaccouse-v3.2-2020EN" />
+        <map value="NL-CM:7.2.9" />
+        <comment value="TypeOfTobaccoUsed" />
+      </mapping>
+    </element>
+    <element id="Observation.component:amount">
+      <path value="Observation.component" />
+      <sliceName value="amount" />
+    </element>
+    <element id="Observation.component:amount.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="266918002" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:amount.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="Amount" />
+      <definition value="The number of cigarettes, cigars or grams of rolling tobacco consumed per day, week, month or year." />
+      <alias value="Hoeveelheid" />
+      <type>
+        <code value="Quantity" />
+      </type>
+      <mapping>
```

Comment:
I wonder if we should fix `.system` on "http;//unitsofmeasure.org/" here. Created a ticket: #85

Reply:
This is resolved by the discussion in the referenced ticket.